docs/online-inference-with-maxtext-engine.md

```diff
@@ -109,6 +109,7 @@ export ICI_TENSOR_PARALLELISM=1
 export SCAN_LAYERS=false
 export WEIGHT_DTYPE=bfloat16
 export PER_DEVICE_BATCH_SIZE=11
+export PROMETHEUS_PORT=9090
```

Comment:
Can we add prometheus metric enabled as a separate example instead of changing the existing config?